test_conformance/relationals/test_shuffles.cpp

```diff
@@ -532,13 +529,21 @@ static int create_shuffle_kernel( cl_context context, cl_program *outProgram, cl
             char progLine2[ 10240 ];
             if( shuffleMode == kBuiltInDualInputFnMode )
             {
-                sprintf( progLine2, shuffleBuiltInDualPattern, get_explicit_type_name( vecType ), inSizeName,
-                        ( inVecSize == 3 ) ? "vload3( %ld, (__global %s *)source )" : "source[ %ld ]",
-                        get_explicit_type_name( vecType ), inSizeName,
-                        ( inVecSize == 3 ) ? "vload3( %ld, (__global %s *)secondSource )" : "secondSource[ %ld ]",
-                        maskPrefix, get_explicit_type_name( maskType ), outSizeName, maskPrefix, get_explicit_type_name( maskType ), outSizeName,
-                        maskString,
-                        ( outVecSize == 3 ) ? "vstore3( tmp, %ld, (__global %s *)dest )" : "dest[ %ld ] = tmp" );
+                sprintf(
+                    progLine2, shuffleBuiltInDualPattern,
+                    get_explicit_type_name(vecType), inSizeName,
+                    (inVecSize == 3) ? "vload3( %ld, (__global %s *)source )"
+                                     : "source[ %ld ]",
+                    get_explicit_type_name(vecType), inSizeName,
+                    (inVecSize == 3)
+                        ? "vload3( %ld, (__global %s *)secondSource )"
+                        : "secondSource[ %ld ]",
+                    maskPrefix, get_explicit_type_name(maskType), outSizeName,
+                    maskPrefix, get_explicit_type_name(maskType), outSizeName,
+                    maskString.c_str(),
```

Comment:
The addition of `.c_str()` is the only real change here; remainder are formatting changes.